src/main/java/org/folio/consortia/service/impl/SharingInstanceServiceImpl.java

```diff
@@ -31,7 +37,7 @@ public SharingInstance getById(UUID consortiumId, UUID actionId) {
     log.debug("getById:: Trying to get by consortiumId: {} and action id: {}", consortiumId, actionId);
     SharingInstanceEntity sharingInstanceEntity = sharingInstanceRepository.findById(actionId).
       orElseThrow(() -> new ResourceNotFoundException("actionId", String.valueOf(actionId)));
-    log.info("getById:: SharedInstance object: {} was successfully retrieved", sharingInstanceEntity.getId());
+    log.info("getById:: sharedInstance object with id: {} was successfully retrieved", sharingInstanceEntity.getId());
```

Comment:
Should there be sharingInstance as in other places in this class?

Reply:
Thank you, corrected.